src/pages-and-resources/discussions/app-list/AppCard.test.jsx

```diff
@@ -1,17 +1,84 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { render, screen } from '@testing-library/react';
+import { Factory } from 'rosie'; // eslint-disable-line import/no-extraneous-dependencies
+
+import AppCard from './AppCard';
+import messages from './messages';
+import '../data/__factories__/index';
+
 describe('AppCard', () => {
+  const partailSupportApp = Factory.build('app');
+  const fullSupportApp = Factory.build('app', { hasFullSupport: true });
+  const selected = true;
+
   test('checkbox input is checked when AppCard is selected', () => {
+    render(
+      <IntlProvider locale="en">
+        <AppCard
+          app={fullSupportApp}
+          onClick={() => jest.fn()}
+          selected={selected}
+        />
+      </IntlProvider>,
+    );
 
+    const radioInput = screen.getByRole('radio');
+    expect(radioInput.getAttribute('aria-checked')).toBe(selected.toString());
+    expect(screen.getByTestId('checkbox').checked).toBe(selected);
   });
 
   test('title, subtitle, and text from app are displayed', () => {
+    const support = partailSupportApp.hasFullSupport
```

Comment:
Why are you checking this variable? The test should be pacific to a scenario. this test should be 

```
title, subtitle, and text from the app are displayed with full support
```
and 
```
title, subtitle, and text from the app are displayed with partial support
```